### ISSUE TITLE
Bulk Issue Creation

### ISSUE THREAD

**Issue by thetoothpick** [pycontribs/jira#209]
Hello,

The JIRA API supports [bulk issue creation](https://docs.atlassian.com/jira/REST/latest/#api/2/issue-createIssue).  Would it be possible to add support for bulk creation to the jira-python package?

The API would look something like this:

``` python
issue_dict_1 = {
    'project': {'id': 123},
    'summary': 'New issue 1 from jira-python',
    'description': 'Look into this one',
    'issuetype': {'name': 'Bug'},
}
issue_dict_2 = {
    'project': {'id': 123},
    'summary': 'New issue 2 from jira-python',
    'description': 'Look into this one too',
    'issuetype': {'name': 'Bug'},
}
new_issue_list = jira.create_issues(field_list=[issue_dict_1, issue_dict_2])
```

**Comment by asqui** [comment]
If you're able to add this functionality and submit a pull request then chances are good that it will be incorporated into the library.

**Comment by thetoothpick** [comment]
I'll see if I can get around to working on it this weekend.

**Comment by thetoothpick** [comment]
This was implemented in 79d9d596bb140d4faaecc84c819e0acb216db319, for the record.
